.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Could you ensure the .env file is not committed? Also, please don't override es-lint.

Reply:
I think I have committed and pushed it as well. So Is anything possible now to uncommit this?

I'll take care of this in future.

Reply:
Remove .env locally + commit to the branch -> In the next commit add ".env" to .gitignore